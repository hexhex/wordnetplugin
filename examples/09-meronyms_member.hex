%% this returns all (member) meronyms of a word
%% POS: nouns

%% the search number for (member) meronyms
%% is 9.
%% Note: In the WordNet documentation this is
%%       the search number for member holonyms.
%%       In fact search 9 fetches words, for
%%       which *word* is a holonym (and these
%%       are its meronyms)

word("pride").
meronyms(W,SW,M,SM) :- &wordnet[9,W](1,SW,M,SM),word(W).
