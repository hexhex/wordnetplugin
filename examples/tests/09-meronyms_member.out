{meronyms("pride",4,"Panthera_leo",1), meronyms("pride",4,"king_of_beasts",1), meronyms("pride",4,"lion",1), word("pride")}
